src/components/interview.ts

```diff
@@ -80,3 +127,80 @@ async function listInterviewers(message: Discord.Message, client: Discord.Client
   outEmbed.setDescription(listString);
   await message.channel.send(outEmbed);
 }
+
+async function clearProfile(message: Discord.Message): Promise<void> {
+  const db = await openDB();
+  const id = message.author.id;
+
+  //clear user data from both tables
+  await db.run('DELETE FROM interviewers WHERE user_id = ?', id);
+  await db.run('DELETE FROM domains WHERE user_id = ?', id);
+  await message.channel.send('Your interviewer data has been cleared!');
+}
+
+async function showProfile(message: Discord.Message): Promise<void> {
+  const db = await openDB();
+  const id = message.author.id;
+
+  //check if user signed up to be interviewer
+  const interviewer = await db.get('SELECT * FROM interviewers WHERE user_id = ?', id);
+  if (!interviewer) {
+    await message.channel.send("You don't seem to have signed up yet, run ```.interviewer signup [link]```");
+    return;
+  }
+
+  //get user's domains (if any)
+  const res = await db.all('SELECT * FROM domains WHERE user_id = ?', id);
+  let domains = '';
+  for (const i of res) {
+    domains += available_domains[i.domain] + ' ';
+  }
+
+  //build output embed
+  const outEmbed = new Discord.MessageEmbed().setColor('#0099ff').setTitle('Interviewer Profile');
+  outEmbed.addField('**Link**', interviewer['link']);
+  outEmbed.addField('**Domains**', domains === '' ? 'None' : domains);
+  await message.channel.send(outEmbed);
+}
+
+async function addDomain(message: Discord.Message, args: string[]): Promise<void> {
+  const db = await openDB();
+  const id = message.author.id;
+
+  //get domain and check arg passed
+  let domain = args.shift();
+  if (!domain) {
+    help(message);
+    return;
+  }
+
+  //parse domain to lowercase and check if valid
+  domain = domain.toLowerCase();
+  if (!(domain in available_domains)) {
+    let output = 'Not a valid domain, valid domains are:';
+    for (const i of Object.values(available_domains)) {
```

Comment:
looks like we built this available domains string a few times, maybe we could refactor this into its own method so we could reuse the same functionality